### PR TITLE
konst week1

### PR DESCRIPTION
**Pull request by gavrilov**
week1

### CODE REVIEW

File: src/konst/week1/finra_konst.java
Comment (cbanavi) [inline src/konst/week1/finra_konst.java:1]: lgtm

File: src/konst/week1/divisionWithoutOperator_konst.java
Comment (cbanavi) [inline src/konst/week1/divisionWithoutOperator_konst.java:1]: lgtm

File: src/konst/week1/oddOrEven_konst.java
Comment (cbanavi) [inline src/konst/week1/oddOrEven_konst.java:1]: lgtm

**Review by cbanavi — APPROVED** [review]
Looks good to me!

File: src/konst/week1/divisionWithoutOperator_konst.java
Comment (jenneft) [reply src/konst/week1/divisionWithoutOperator_konst.java:1]: lgtm

File: src/konst/week1/finra_konst.java
Comment (jenneft) [reply src/konst/week1/finra_konst.java:1]: great

File: src/konst/week1/oddOrEven_konst.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+public class oddOrEven_konst {
+    public static void checkEvenOdd(int number) {
+        System.out.println(number % 2 == 0 ? "even" : "odd");
+    }
+
+    public static void main(String[] args) {
+        checkEvenOdd(12);
+    }
+}
```
Comment (jenneft) [inline src/konst/week1/oddOrEven_konst.java:13]: looks good

**Review by jenneft — APPROVED** [review]
Approved

**Review by bernasantiago — COMMENTED** [review]
lgtm!

**Pull request merged**